examples/apps/spaces/src/fluid-object/storage/spacesStorageView.tsx

```diff
@@ -21,7 +23,7 @@ const SpacesEditButton: React.FC<ISpacesEditButtonProps> =
     (props: React.PropsWithChildren<ISpacesEditButtonProps>) =>
         <button
             className="spaces-edit-button"
-            onClick={props.clickCallback}
+            onClick={props.clickCallback.bind(this)}
```

Comment:
Similar here and 3 more below -- I don't think you ever want to bind in a React FC